xonsh/tools.py

```diff
@@ -911,25 +911,66 @@ def suggest_commands(cmd, env, aliases):
     return rtn
 
 
-def print_exception(msg=None):
-    """Print exceptions with/without traceback."""
+def _get_env_var(name, default=None):
```

Comment:
maybe include something in the function name here about being manually set to make it more clear

Reply:
ie `_get_manual_env_var`

Reply:
Fixed in a5b0993